Sources/ArgumentsUtils.swift

```diff
@@ -126,20 +134,24 @@ let bundledTemplatesPath = Path(ProcessInfo.processInfo.arguments[0]).parent() +
  - returns: The Path matching the template to find
  */
 func findTemplate(prefix: String, templateShortName: String, templateFullPath: String) throws -> Path {
-  guard templateFullPath.isEmpty else {
+  switch (templateFullPath, templateShortName) {
+  case ("", ""): throw TemplateError.noTemplateFound
```

Comment:
Maybe we should just change the error message [here](https://github.com/SwiftGen/SwiftGen/blob/aaa5dd3ff1f0fb166f24be5fddee00e8fe4761cd/Sources/ArgumentsUtils.swift#L93-L94) for this?